bin/dgVoodooCpl.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/env bash
+
+export C=$WINEPREFIX/drive_c
+export PREFIX_DGV=$C/launcher/client/dgVoodooCpl.exe
```

Comment:
I guess this should be no longer needed as well?

Reply:
This part is still needed, as we will be shipping that executable in our upstream as mentioned above

Reply:
no, our updated game client files, this flatpak is just the runtime env and launcher that keeps it hydrated, that's why I was saying we don't put the whole game files in an immutable OCI layer- it also can't be immutable/read-only.

Reply:
so whatever the client downloads?

Reply:
Yes. Do you scrutinize steam the same way? Because that's the same pattern here.